src/NoFrixion.MoneyMoov/Permissions/Permissions.cs

```diff
@@ -114,9 +114,9 @@ public enum MerchantPermissions : ulong
     CanCreateMandates = 8589934592,
     
     // Permissions
-    CanViewGroups = 17179869184,
-    CanCreateGroups = 34359738368,
-    CanEditGroups = 68719476736,
+    CanViewRoles = 17179869184,
+    CanCreateRoles = 34359738368,
+    CanEditRoles = 68719476736,
 }
```

Comment:
What do you think about using left shift to make this more readable and less error prone (1 digit wrong and ti could break):

    CanViewRoles = 1L << 34, 
    CanCreateRoles = 1L << 35, 
    CanEditRoles = 1L << 36

Reply:
Yeah good idea